### DISCUSSION TITLE
리드잇zine 5호 나왔습니다.

### DISCUSSION THREAD

**Discussion by jongfeel** [ThinkAboutSoftware/AcademicConference#235] [General]
예전에 태형님이 올려 주셨던 

- #203 

요기에 원고를 보내서 실리게 되었습니다.

http://www.kyobobook.co.kr/eventRenewal/eventViewByPid.laf?eventPid=90507&orderClick=LCF

종이책으로 출판된건 판매는 안하는 것 같고 잡지 기고한 사람들한테만 배부하는 것 같아요
어제 10부 택배로 받았습니다.

한두달 정도 후면 ebook도 나올거 같은데
그때 보시면 될 것 같습니다. 무료입니다.

**Comment by TaeHyoungKwon** [comment]
기대됩니다 ebook 나오면 꼭 보겠습니다~ 😄

**Reply by jongfeel** [reply]
감사합니다!
